lightning/src/ln/channelmanager.rs

```diff
@@ -7202,6 +7207,133 @@ where
 	}
 }
 
+const BOLT_12_INVOICE_RETRY_STRATEGY: Retry = Retry::Attempts(3);
+
+impl<M: Deref, T: Deref, ES: Deref, NS: Deref, SP: Deref, F: Deref, R: Deref, L: Deref>
+OffersMessageHandler for ChannelManager<M, T, ES, NS, SP, F, R, L>
```

Comment:
Could we put this in the `offers` module to avoid adding more stuff to `channelmanager.rs`?